index.html

```diff
@@ -1,14 +1,50 @@
-Diana Carachure
-About
-Diana has 10 years of experience in the nonprofit sector. Diana has worked in both direct client services and administration roles.
-She holds bachelor's in psychology and is currently enrolled in Code the Dream's intro to programming class. 
-Experience 
-Advancement Specialist May 2021- present
--contracts billing, fundraiser support, quality assurance, etc
-Data 
--kkn;ek
-Lead Case Manager
--ikjfeqj;
-Connect
-Email
-Github
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Welcome to my site!</title>
+    </head>
+    
+    <body>
+        <div>
+        <nav>
+            <h1>Diana Carachure</h1>
+            <p>New to coding! Follow along my journey!</p>
+            <ul>
+                <li><a href="/#/about">About></a></li>
+                <li><a href="/#/experience">Experience></a></li>
+                <li><a href="/#/connect">Connect></a></li>
+            </ul>
+        </nav>
+
+    <section id="about">
+        <h2>About</h2>
+            <p>Diana has 10 years of experience in the nonprofit sector. Diana has worked in both direct client services and administration roles.
```

Comment:
Even though it seems like the `<p>` element should be "under" the `<h2>`, they are actually sibling elements (at the same level of nesting), so they should be indented at the same level.